core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -411,8 +413,8 @@ trait EntityComponent {
       EntityRecordRawSqlQuery.activeActionForRefs(workspaceId, entities)
     }
 
-    private def findActiveAttributesByEntityId(workspaceId: UUID, entityId: Rep[Long]): EntityAttributeQuery = for {
-      entityAttrRec <- entityAttributeShardQuery(workspaceId) if entityAttrRec.ownerId === entityId && ! entityAttrRec.deleted
+    private def findActiveAttributesByEntityId(workspaceId: UUID, sharded: Option[Boolean], entityId: Rep[Long]): EntityAttributeQuery = for {
```

Comment:
most of the changes in this file past this point are just changing signatures to weave through the new `sharded` value